linkerd/service-profiles/src/client.rs

```diff
@@ -224,15 +224,15 @@ where
                     }
                     Err(e) => {
                         warn!("error fetching profile: {:?}", e);
-                        State::Backoff(Delay::new(clock::now() + self.backoff))
+                        State::Backoff(Delay::new(Instant::now() + self.backoff))
```

Comment:
At one point, I believe we were trying to use `tokio_timer::clock::now()` rather than `Instant::now()` everywhere. Is this something we still care about? IIRC, we aren't doing any large-scale integration testing with the mock clock...are there other reasons to prefer `clock::now()`?

Reply:
I think it only makes sense in contexts where we are actually mocking time. We generally don't do that.